sbncode/CAFMaker/FillReco.cxx

```diff
@@ -557,7 +558,16 @@ namespace caf
     if (t0match.size()) {
       assert(t0match.size() == 1);
       srtrack.crthit.distance = t0match[0]->fTriggerConfidence;
+      srtrack.crthit.region = t0match[0]->fID;
+      srtrack.crthit.sys = t0match[0]->fTriggerBits;      
+      srtrack.crthit.deltaX = hitmatchinfo[0]->DeltaX;
```

Comment:
This would crash len(hitmatchinfo) == 0 and len(t0match) == 1, as I believe would be the case in SBND.

Could you change this to include a separate if statement of: if (hitmatchinfo.size())